boilerplate/openshift/golang-lint/ensure.sh

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env bash
```

Comment:
Currently an exact copy of the one from golang-osd-operator -- but agree with comment below that it should be stripped down to just the lint-related bits.